napari/components/text_overlay.py

```diff
@@ -25,7 +31,16 @@ class TextOverlay(EventedModel):
 
     # fields
     visible: bool = False
-    color: str = "#FFFFFF"
+    color: Optional[Array[float, (4,)]] = (1.0, 1.0, 1.0, 1.0)
```

Comment:
I know I copied and pasted the optional over here, but looking at this again - I think we don't need the optional - there should always be a text color right? We could then drop the return `None` stuff and just have the `transform_color(v)[0]` I think

Reply:
Nope, that makes sense - it's been corrected now.